src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -7,18 +7,144 @@
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+
+    private Entry<K, V>[] table;
+    private double threshold;
```

Comment:
this variable seems to be redundant, you use it only once as size / table.length

Reply:
done